add-chain/go.mod

```diff
@@ -0,0 +1,33 @@
+module github.com/ethereum-optimism/superchain-registry/add-chain
+
+go 1.21.9
```

Comment:
@bitwiseguy shall we harmonize the `go` version here across all modules to `1.21` for now?

Reply:
Fixed in https://github.com/ethereum-optimism/superchain-registry/pull/204/commits/1b2967aaae1057c94f408e931162cbcf28a2b043